app/scripts/models/user.js

```diff
@@ -524,6 +530,19 @@ define(function (require, exports, module) {
           }
           return exists;
         });
+    },
+
+    beginFlow: function (flowBeginTime) {
+      if (! this.has('flowId')) {
```

Comment:
I could imagine [one of the first things in flow.js->begin](https://github.com/mozilla/fxa-content-server/pull/3619/files#diff-0a6ffa0f4795903fe29afcfc1e11ff95R98) would be:

```js
if (this.has('flowId')) {
  // flowId already set, no need to re-initialize.
  return;
}
```

Though I realize I do not have full context.